starter_code/stylesheets/style.css

```
@@ -8,3 +8,166 @@ paragraph yellow: #ECB12F
 links blue: #2E71A6
 footer links grey: #454245
 */
+
+html, body {
+    margin: 0;
+    padding: 0;
+}
+
+
```

So in order to deal with the images at the start you can set them to a specific width, then depending on the image you can add an special class an modify the width
```suggestion
img {
width: 25%;
}

```